### PR TITLE
made fx public

### PR DESCRIPTION
**Pull request by williamFalcon**
fixes potentially missing fx

### CODE REVIEW

File: pytorch_lightning/trainer/distrib_data_parallel.py [hunk 1]
```diff
@@ -369,7 +369,7 @@ def set_nvidia_flags(self, is_slurm_managing_tasks, data_parallel_device_ids):
         # don't make this debug... this is good UX
         rank_zero_info(f'CUDA_VISIBLE_DEVICES: [{os.environ["CUDA_VISIBLE_DEVICES"]}]')
 
-    def __set_random_port(self):
+    def set_random_port(self):
```
Comment (Borda) [inline pytorch_lightning/trainer/distrib_data_parallel.py:372]: I would keep it protected >> just one '_'

File: docs/source/trainer.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ Trainer
    :exclude-members:
         run_pretrain_routine,
         _abc_impl,
-        _Trainer__set_random_port,
+        set_random_port,
```
Comment (Borda) [inline docs/source/trainer.rst:12]: is is still Trainer method
```suggestion
        _Trainer_set_random_port,
```

Comment (Borda) [reply docs/source/trainer.rst:12]: but maybe you are right, need to see build docs

File: pytorch_lightning/trainer/distrib_data_parallel.py (hunk 1 above)
Comment (Borda) [reply pytorch_lightning/trainer/distrib_data_parallel.py:372]: private has an issue with inheritance but protected is fine :rabbit:

File: docs/source/trainer.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ Trainer
    :exclude-members:
         run_pretrain_routine,
         _abc_impl,
-        _Trainer__set_random_port,
+        _Trainer_set_random_port,
         _Trainer__set_root_gpu,
         _Trainer__init_optimizers,
         _Trainer__parse_gpu_ids,
```
Comment (Borda) [inline docs/source/trainer.rst:15]:
```suggestion
```
can be removed as non of them is presented in the docs

Comment (williamFalcon) [reply docs/source/trainer.rst:15]: they're not presented in the docs because they are listed here lol...

Comment (Borda) [reply docs/source/trainer.rst:15]: even they are not listed here I have not found them in docs

**Pull request merged**